src/main/java/pages/ChooseDiscussionPage.java

```diff
@@ -0,0 +1,25 @@
+package pages;
+
+import elements.CustomButton;
+import io.qameta.allure.Step;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.support.FindBy;
+
+public class ChooseDiscussionPage extends BasePage {
+
+    private static final Logger logger = Logger.getLogger("Choose discussion page logger");
+
+    @FindBy(xpath = "//*[contains(@class, 'topictitle') and text()='Задай вопрос A1']")
+    private CustomButton chooseDiscussionButton;
+
+    public ChooseDiscussionPage() {
+        super();
+    }
+
+    @Step("Choose discussion")
+    public DiscussionPage chooseDiscussion() {
```

Comment:
the method name is not clear. 'Choose discussion' is different from 'click choose discussion button'